apache-spark-worker/build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+RUN apt-get update && \
+  apt-get install -y \
+    python \
+    libnss3 \
+    openjdk-8-jdk \
+    curl
+
+COPY download.sh /home/
+RUN sh /home/download.sh
+
+WORKDIR /opt/apache-spark
+RUN sed -i "s/<joda.version>2.5<\/joda.version>/<joda.version>2.9.4<\/joda.version>/" pom.xml
```

Comment:
It's the build container so adding a dependency is not a problem. Just be aware that xmlstarlet might also succeed without making a substitution. And it may change the formatting of the pom.xml. But that should not matter for the build unless the pom has scripts embedded into it.